models/engine/file_storage.py

```diff
@@ -68,3 +68,19 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        ''' Retrieves an object'''
+        for object in self.all(cls).values():
+            if object.id == id:
+                return object
+
+        return None
+
+    def count(self, cls=None):
+        ''' counts the number of objects in storage '''
+        count = 0
+        for object in self.all(cls).values():
```

Comment:
Check if your code colapse by recieving `Pichu` as class

Reply:
Good observation, will take into account in unittests